### DISCUSSION TITLE
How to constrains some atoms in a MD run?

### DISCUSSION THREAD

**Discussion by icamps** [grimme-lab/xtb#1113] [Q&A]
I need to constrain the positions of some atoms in a molecular dynamic run.

I used the following input, unsuccessfully (the atoms move):

```
$md
   temp=298.15 # in K
   time= 20.0  # in ps
   dump= 50.0  # in fs
   step=  2.0  # in fs
   velo=false
   nvt =true
   hmass=4
   shake=2
   sccacc=2.0
$constrain
   atoms: 3,23,33,63,73,93,8,28,38,68,78,98
$end
```

I read somewhere in the xTB documentation that it is not recommended to use constrains in MD calculations, but I need to try it.

Regards,
Camps

**Comment by marcelmbn** [comment]
Have you read the corresponding documentation in https://xtb-docs.readthedocs.io/en/latest/xcontrol.html#constraining-potentials?
